lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -50,6 +51,8 @@ $(document).ready(function() {
   // Initialize the donation form on the donate page.
   Donate.init();
 
+  Globe.init();
```

Comment:
Please add a comment explaining what this module does (see the other ones for examples).